float32 nP
float32 nI
float32 nD
---
float32 nResult

